docs/source/variational-problems.rst

```diff
@@ -288,15 +288,47 @@ For discontinuous elements these are the Gauss-Legendre points, and for
 continuous elements these are the Gauss-Lobatto-Legendre points.
 For CG and DG spaces on simplices, Firedrake offers both equispaced points and
 the better conditioned recursive Legendre points from :cite:`Isaac2020` via the
-`recursivenodes`_ module. These are selected by passing `variant="equispaced"`
-or `variant="spectral"` to the :py:class:`~ufl.classes.FiniteElement` or
+`recursivenodes`_ module. These are selected by passing ``variant="equispaced"``
+or ``variant="spectral"`` to the :py:class:`~finat.ufl.finiteelement.FiniteElement` or
 :py:func:`~.FunctionSpace` constructors. For example:
 
 .. code-block:: python3
 
     fe = FiniteElement("RTCE", quadrilateral, 2, variant="equispaced")
 
-The default is the spectral variant.
+For CG and DG spaces, and most tensor-product elements, the default is ``variant="spectral"``.
+
+Integral-type degrees of freedom are also available through ``variant="integral"``. These
+enable orthogonal bases for DG on any cell type:
+
+.. code-block:: python3
+
+    V = FunctionSpace(mesh, "DG", 1, variant="integral")
+
+In this case, the degrees of freedom are integral moments against a basis of polynomials,
+which need to be computed with a quadrature rule. The degree of accuracy of the quadrature
+can be increased also through the variant option. For instance, ``variant="integral(4)"``
```

Comment:
`variant` and the quadrature rule bits are clearly somewhat related. Should they exist on the same manual page? Or at least link to each other?